e2e/helpers/docker_compose.py

```diff
@@ -0,0 +1,62 @@
+import os
+import shlex
+import subprocess
+from time import sleep
+
+
+def stop():
+    subprocess.check_call("docker-compose stop -t 0".split(' '))
+
+
+def clean():
+    stop()
+    subprocess.check_call("docker-compose rm -vf".split(' '))
+
+
+def build():
+    subprocess.check_call("docker-compose build".split(' '))
+
+
+def pull():
+    subprocess.check_call("docker-compose pull".split(' '))
+
+
+def start():
+    subprocess.check_call("docker-compose up -d databasedev".split(' '))
+
+    sleep(10)
+
+    subprocess.check_call(
+        shlex.split("docker-compose run --rm webappdev /bin/bash -c './manage.py migrate'", comments=True)
+    )
+
+    subprocess.check_call("docker-compose up -d".split(' '))
+
+
+def create_superuser_for_test(username, password, email=""):
+    """
+    password is `password` for testing purposes
+    """
+    superuser_command = """from django.contrib.auth.models import User
+User.objects.create_superuser(username='{username}', password='{password}', email='{email}')
+    """.format(email=email, username=username, password=password)
+    # TODO: this only works on development mode with mounted source volumes
+    filename = os.path.join(os.path.dirname(__file__), '..', '..', 'osmaxx-py', 'create_superuser.py')
+    file = open(filename, mode='w')
+    file.write(superuser_command)
```

Comment:
Do we have to store this python code in a file to run it within Django (`manage.py`) context? I think I'd prefer piping it into `python manage.py shell`. (Python has quite some capabilities to manage pipes.)